main/output.c

```diff
@@ -478,7 +478,7 @@ PHPAPI php_output_handler *php_output_handler_create_user(zval *output_handler,
 		default:
 			user = ecalloc(1, sizeof(php_output_handler_user_func_t));
 			if (SUCCESS == zend_fcall_info_init(output_handler, 0, &user->fci, &user->fcc, &handler_name, &error)) {
-				handler = php_output_handler_init(handler_name, chunk_size, (flags & ~0xf) | PHP_OUTPUT_HANDLER_USER);
+				handler = php_output_handler_init(handler_name, chunk_size, (flags & ~0xf00f) | PHP_OUTPUT_HANDLER_USER);
```

Comment:
Could this maybe not be a magic value? Maybe ``(flags & ~0xf00f)`` should be a macro function for clarity?